examples/notebooks/TextCNN.ipynb

```diff
@@ -997,7 +997,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.8"
+   "version": "3.11.1"
```

Comment:
Since your changes have nothing to do with the notebook contents' required python version, no need to change this as well.